README.md

```diff
@@ -1,52 +1,64 @@
 # susemanager-ci
 
-CI Automation for [SUSE Manager](https://www.suse.com/products/suse-manager/) and [Uyuni](https://www.uyuni-project.org/)
+CI automation for [SUSE Manager](https://www.suse.com/products/suse-manager/) and [Uyuni](https://www.uyuni-project.org/).
```

Comment:
as @srbarrios commented, in fact we are doing CT, not CI...

but maybe that's for a separate PR